python/paddle/fluid/tests/custom_op/extension_utils.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import six
+import sys
+import copy
+import glob
+import warnings
+import subprocess
+
+import paddle
+
+IS_WINDOWS = os.name == 'nt'
+# TODO(Aurelius84): Need check version of gcc and g++ is same.
+# After CI path is fixed, we will modify into cc.
+NVCC_COMPILE_FLAGS = [
+    '-ccbin', 'gcc', '-DPADDLE_WITH_CUDA', '-DEIGEN_USE_GPU',
+    '-DPADDLE_USE_DSO', '-Xcompiler', '-fPIC', '-w', '--expt-relaxed-constexpr',
+    '-O3', '-DNVCC'
+]
```

Comment:
This Saturday，tianshuo will fix cc path problem on our PY3-CI  environment. After his work, I will modify `gcc` into `cc` to unify command of compiler.